pkg/kubelet/kubelet.go

```diff
@@ -598,7 +599,7 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		HairpinMode:        kubeletconfiginternal.HairpinMode(kubeCfg.HairpinMode),
 		NonMasqueradeCIDR:  nonMasqueradeCIDR,
 		PluginName:         crOptions.NetworkPluginName,
-		PluginConfDir:      crOptions.CNIConfDir,
+		PluginConfDirs:     cni.SplitDirs(crOptions.CNIConfDir),
```

Comment:
if CNIConfDir is a multi-value option, can we make it a `[]string`, not plumb down to an arbitrary point and then split on comma?